crates/modelardb_server/src/storage/data_transfer.rs

```diff
@@ -265,6 +266,7 @@ mod tests {
     use std::path::Path;
 
     use chrono::Utc;
+    use modelardb_common::metadata::try_new_sqlite_table_metadata_manager;
```

Comment:
Maybe only import `modelardb_common::metadata` and use the function as  `metadata::try_new_sqlite_table_metadata_manager()` for consistency.

Reply:
Fixed.